input/fsh/moped-operation-verlegen.fsh

```diff
@@ -21,7 +21,7 @@ Die Operation wird vom Akteur Krankenhaus (KH) aufgerufen. Die $verlegen Operati
   * *MOPEDTransferEncounter.serviceProvider* setzt eine Referenz auf die MOPEDOrganizationAbteilung mit dem jeweiligen *funktionscode* bzw. *funktionssubcode* lt. Operation-Parameter.
   * *MOPEDTransferEncounter.Neugeborenes* wird lt. LKF-Regeln berechnet, anhand des *MOPEDEncounter.subject.birthdate* aus dem Encounter aus Schritt 1 (für Berechnugns-Details siehe Hinweis 1).
   * *MOPEDTransferEncounter.Altersgruppe* wird lt. LKF-Regeln berechnet, anhand des *MOPEDEncounter.subject.birthdate* aus dem Encounter aus Schritt 1 (für Berechnugns-Details siehe Hinweis 2).
-  * *MOPEDTransferEncounter.PhysischeAnwesenheit* wird lt. Operation-Parameter befüllt.
+  * *MOPEDTransferEncounter.anwesenheitsart* enthält wert aus dem ValueSet *MOPEDTransferEncounter.subjectStatus.coding.display*
```

Comment:
also es sollte in dem Fall nicht nur `coding.display` befüllt werden, sondern auch `coding.code`. In der Operation mitgegeben wird nur `coding.code` aber den Display kann man über das Value-Set-Binding herausfinden.

Reply:
Hab die Beschreibung nochmal geupdated.